app/helpers/emails_helper.rb

```diff
@@ -31,8 +31,8 @@ def action_title(url)
   end
 
   def color_email_diff(diffcontent)
-    formatter = Rugments::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
-    lexer = Rugments::Lexers::Diff.new
+    formatter = Rouge::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
```

Comment:
Thanks for the review, @jneen. I downloaded `rouge-1.9.1.gem` from RubyGems, ran `gem unpack` on it, and then did this:

```
$ find . -name html_inline.rb
./rouge-1.9.1/lib/rouge/formatters/html_inline.rb
```

Is this some packaging issue?